tsl/test/expected/bgw_db_scheduler.out

```diff
@@ -168,21 +168,22 @@ SELECT ts_bgw_db_scheduler_test_run_and_wait_for_scheduler_finish(50);
 
 \x on
 SELECT * FROM _timescaledb_internal.bgw_job_stat;
--[ RECORD 1 ]----------+--------------------------------
-job_id                 | 1000
-last_start             | Fri Dec 31 16:00:00.05 1999 PST
-last_finish            | Fri Dec 31 16:00:00.05 1999 PST
-next_start             | Fri Dec 31 16:00:00.15 1999 PST
-last_successful_finish | Fri Dec 31 16:00:00.05 1999 PST
-last_run_success       | t
-total_runs             | 1
-total_duration         | @ 0
-total_successes        | 1
-total_failures         | 0
-total_crashes          | 0
-consecutive_failures   | 0
-consecutive_crashes    | 0
-flags                  | 0
+-[ RECORD 1 ]-----------+--------------------------------
+job_id                  | 1000
+last_start              | Fri Dec 31 16:00:00.05 1999 PST
+last_finish             | Fri Dec 31 16:00:00.05 1999 PST
```

Comment:
wont this introduce flakyness on slower ci environments when the job cant be finished in those 0.05 seconds

Reply:
I don't think so, because in those tests we use a virtual timer that advances when mock_wait is called, and the setting for mock_wait in this case is `WAIT_ON_JOB` (https://github.com/timescale/timescaledb/blob/2475c1b92ff9e566cb3858924ca153aa14ab4a25/test/src/bgw/timer_mock.c#L53) 
The handle on which we wait is the handle of the last registered job, in this case 1000, so the job will complete first before advancing the timer.